### PR TITLE
docs(graphql): add csrf error solution

### PR DESCRIPTION
**Pull request by Shivam1337**
#### Docs
- Added Apollo CSRF Error Solution For MultipartFile Upload

### CODE REVIEW

File: packages/graphql/README.md [hunk 1]
```diff
@@ -281,6 +281,18 @@ final result = await graphQLClient.mutate(
...
+<br>
+Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`
+
+```dart
+HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
+  'Content-Type': 'application/json; charset=utf-8',
+  'X-Apollo-Operation-Name': 'post'
+})
+```
+
```
Comment (vincenzopalazzo) [inline packages/graphql/README.md:295]: We can put this in section # Q&A  with the following format
```suggestion
## Q&A

*CSRF Error:* If you are receiving csrf error from your apollo graphql server while uploading file,
your need to add some additional headers to the `HttpLink`:
<br>
Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`

```dart
HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
  'Content-Type': 'application/json; charset=utf-8',
  'X-Apollo-Operation-Name': 'post'
})
```
- I'm receiving a `CSRF Error` from apollo graphql server while uploading file, what I need to do?
Add some additional headers to the `HttpLink`:
<br>
Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`

```dart
HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
  'Content-Type': 'application/json; charset=utf-8',
  'X-Apollo-Operation-Name': 'post'
})
```
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
@budde377 what do you think to make a Q&A section?

**Comment by vincenzopalazzo** [comment]
@Shivam1337 you should move the configuration to the bottom of the readme with # Q&A

Please not fixed the code block format, and squash the commit in a single one!

Thanks!

**Review by vincenzopalazzo — APPROVED** [review]
awesome!

**Review by budde377 — APPROVED** [review]

**Pull request merged**